core/settings/common.py

```diff
@@ -0,0 +1,280 @@
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+# update
+
+import os
+import json
+
+import environ
+
+from .dependencies.ckeditor import CKEDITOR_5_CONFIGS, CKEDITOR_5_CUSTOM_CSS  # noqa
+
+
+env = environ.Env(
+    # set casting, default value
+    DEBUG=(bool, False),
+    DEVELOP=(bool, False),
+)
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))  # TODO Fix this
+
+# Quick-date development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = env('SECRET_KEY', str, 'SECRET_KEY')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = env('DEBUG', bool, False)
+
+DEVELOP = env('DEVELOP', bool, False)
+
+# This gets set only when tests are ran with date-test command
+TEST = env('TEST', bool, False)
+
+ALLOWED_HOSTS = json.loads(env('ALLOWED_HOSTS', str, '[]'))
+
+CSRF_TRUSTED_ORIGINS = json.loads(env('ALLOWED_ORIGINS', str, '[]'))
+
+
+def get_installed_apps(proj_apps):
+    return [
+        'daphne',
+        'date',
+        'members',
+        *proj_apps,
+        'django.contrib.admin',
+        'django.contrib.auth',
+        'django.contrib.contenttypes',
+        'django.contrib.sessions',
+        'django.contrib.messages',
+        'django.contrib.staticfiles',
+        'admin_ordering',
+        'django_ckeditor_5',
+        'channels',
+        'storages',
+        'django_tables2',
+        'django_filters',
+        'bootstrap3',
+        'django_cleanup',  # Should be places last
+    ]
+
+
+MIDDLEWARE = [
+    'whitenoise.middleware.WhiteNoiseMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+
+    'date.middleware.LangMiddleware',
+    'date.middleware.HTCPCPMiddleware',
+]
+
+
+WSGI_APPLICATION = 'core.wsgi.application'
+ASGI_APPLICATION = 'core.routing.application'
+
+
+CHANNEL_LAYERS = {
+    'default': {
+        'BACKEND': 'channels_redis.core.RedisChannelLayer',
+        'CONFIG': {
+            "hosts": [('redis', 6379)],
+        },
+    },
+}
+
+
+# Database
+# https://docs.djangoproject.com/en/2.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'postgres',
+        'USER': 'postgres',
+        'PASSWORD': env('DB_PASSWORD', default=''),
+        'HOST': 'db',
+        'PORT': 5432,
```

Comment:
These could maybe be env vars?

Reply:
Still need to go over this several times, so very much WIP for now :D